src/MappingBar/zoneSearchUtils.ts

```diff
@@ -0,0 +1,42 @@
+import startsWith from 'lodash/startsWith'; // lodash is faster than native implementation
+
+import { FilterOptionsState } from '@material-ui/lab/useAutocomplete';
+
+export interface ZoneLight {
+  name: string;
+  value: string;
+}
+
+export const filterZones = (
+  zoneList: ZoneLight[],
+  state: FilterOptionsState<ZoneLight>
+) => {
+  const inputVal: string = state.inputValue.toLowerCase();
+
+  const newZoneList = zoneList.filter((z) => startsWith(z.value, inputVal));
+
+  if (newZoneList.length) {
+    return newZoneList;
+  }
+
+  const inputTerms = inputVal.split(' ');
+
+  return inputTerms.reduce(
+    (list: ZoneLight[], term) => list.filter((i) => i.value.indexOf(term) >= 0),
```

Comment:
The term should be made into lowercase to match the values on ZoneLight, that should fix the issue you commented about.

Reply:
@Tebro that's the point in ZoneLight, value is lower case already. 


https://github.com/Portaler-Zone/albion-mapper/pull/7/files#diff-e2c6f314e3be1a6d210c4b836954daf7120eae1ed14942046c1f1b353d244981R46